README.md

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Deploy
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
```suggestion
      - uses: actions/checkout@v3.4.0
```

It can be an even more up-to-date version as well

Reply:
v3 should take the latest v3.x.x

Reply:
That is absolutely correct! Disregard my message in this case :)